docs/tutorial/pre-requisites-of-running-on-chain-spam-protected-chat2.md

```diff
@@ -16,96 +19,89 @@ If you already have an Ethereum account with sufficient ethers on the Sepolia te
-     [https://sepolia-faucet.pk910.de/](https://sepolia-faucet.pk910.de/)
-6. Enter your Eth account address, accept Captcha and start mining.
-    
-    ![](https://i.imgur.com/IvHNsei.jpg)
+## 2. Obtain Sepolia ETH from faucet
+
+Sepolia ETH can be obtained from different faucets.
+Three popular examples include:
+
+  1. [sepoliafaucet.com](https://sepoliafaucet.com/) (requires an Alchemy account)
+  2. [Infura Sepolia faucet](https://www.infura.io/faucet/sepolia) (requires an Infura account)
+  3. [Sepolia POW faucet](https://sepolia-faucet.pk910.de/)
+
+> _*NOTE:*_ This list is provided for convenience. We do not necessarily recommend or guarantee the security of any of these options.
 
-7. You can see the estimated Sepolia ETH mined per hour. Each session is restricted to a few hours.
+Many faucets limit the amount of Sepolia ETH you can obtain per day.
+To obtain more, a faucet based on PoW mining such as [sepolia-faucet.pk910.de](https://sepolia-faucet.pk910.de/) can be used:
+
+1. Enter your Sepolia Ethereum account public address, accept the Captcha and start mining.
     
-    ![](https://i.imgur.com/qTWwyNr.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/8bf2eece-956c-4449-ac4c-a7b9f4641c99)
 
-8. When you exceed the hour limit of the session, then the mining will be stopped.
-9. Alternatively, stop the mining when mined enough sepolia ether.
-10. Do not forget to claim your sepolia ether.
+2. Keep the browser tab open for a while. You can see the estimated Sepolia ETH mined per hour. 
     
-    ![](https://i.imgur.com/cGCSexB.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/fac1c6cb-b72f-47b1-a358-4ce41224a688)
+
+   Each session is limited to a few hours. 
+3. When you've mined enough Sepolia ETH (minimum of 0.05 Sepolia ETH), click on "Stop Mining" and claim your reward.
     
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/9ace2824-9030-4507-9b5f-50354bb99127)    
     
-Setting up a hosted node on Sepolia Testnet using Infura
----
+## 3. Setting up a hosted node on Sepolia Testnet using Infura
 
-(Note: Infura provides a simple straight-forward way of setting up endpoints for interaction with the Ethereum chain and the Waku RLN smart contract without having to run a dedicated Ethereum node. Setting up infura is not mandatory. Operators concerned with the centralized aspect introduced by infura can setup their own node.)
+> _*NOTE:*_ Infura provides a simple way of setting up endpoints for interaction with the Ethereum chain and the Waku RLN smart contract without having to run a dedicated Ethereum node.
+> Setting up Infura is not mandatory. Operators concerned with the centralized aspect introduced by Infura should use their own node.
 
-1. Sign up to infura if you do not have an account already. [https://infura.io/register](https://infura.io/register)
+1. Sign up to Infura if you do not have an account already. [https://infura.io/register](https://infura.io/register)
```

Comment:
> Sign up to

Sign up for?

Reply:
Fixed in https://github.com/waku-org/nwaku/pull/2206/commits/4d921d9426bd22664ee3cf19ea355fb1219fe890